source/operations/checklists/software.rst

```diff
@@ -8,4 +8,71 @@ Software Checklist
    :local:
    :depth: 2
 
-Stub - detail software checklist items
+Use the following checklist when planning the software configuration for a production, distributed MinIO deployment.
+
+MinIO Pre-requisites
+--------------------
+
+.. list-table::
+   :widths: auto
+   :width: 100%
+
+   * - :octicon:`circle`
+     - Servers running a Linux operating system with a 5.x+ kernel, such as Red Hat Enterprise Linux (RHEL) 9 or Ubuntu LTS 18.04+
```

Comment:
:grimacing: let's not encourage use of 18.04. Also, 18.05 does not include a 5.x kernel by default

Reply:
Upped to 20.04+